sql/mysql/driver.go

```diff
@@ -282,7 +282,7 @@ const (
 	defaultGen    = "default_generated"
 	autoIncrement = "auto_increment"
 
-	genVirtual    = "virtual"
-	genStored     = "stored"
-	genPersistent = "persistent"
+	genVirtual    = "VIRTUAL"
+	genStored     = "STORED"
+	genPersistent = "PERSISTENT"
```

Comment:
Why `gen` prefix?

Reply:
No special reason. I love grouping/namespacing constants that are related to each other. I can remove the prefix.